include/language/en_us.lang.php

```diff
@@ -1197,41 +1121,28 @@
     'LBL_EMAIL_MULT_GROUP_FOLDER_ACCOUNTS' => 'Multiple',
     'LBL_EMAIL_MULT_GROUP_FOLDER_ACCOUNTS_EMPTY' => 'Empty',
     'LBL_EMAIL_DATE_SENT_BY_SENDER' => 'Date Sent by Sender',
-    'LBL_EMAIL_DATE_RECEIVED' => 'Date Received',
-    'LBL_EMAIL_ASSIGNED_TO_USER' => 'Assigned to User',
     'LBL_EMAIL_DATE_TODAY' => 'Today',
-    'LBL_EMAIL_DATE_YESTERDAY' => 'Yesterday',
-    'LBL_EMAIL_DD_TEXT' => 'email(s) selected.',
-    'LBL_EMAIL_DEFAULTS' => 'Defaults',
     'LBL_EMAIL_DELETE' => 'Delete',
     'LBL_EMAIL_DELETE_CONFIRM' => 'Delete selected messages?',
     'LBL_EMAIL_DELETE_SUCCESS' => 'Email deleted successfully.',
     'LBL_EMAIL_DELETING_MESSAGE' => 'Deleting Message',
     'LBL_EMAIL_DETAILS' => 'Details',
-    'LBL_EMAIL_DISPLAY_MSG' => 'Displaying email(s) {0} - {1} of {2}',
-    'LBL_EMAIL_ADDR_DISPLAY_MSG' => 'Displaying email address(es) {0} - {1} of {2}',
 
-    'LBL_EMAIL_EDIT_CONTACT' => 'Edit Contact',
     'LBL_EMAIL_EDIT_CONTACT_WARN' => 'Only the Primary address will be used when working with Contacts.',
-    'LBL_EMAIL_EDIT_MAILING_LIST' => 'Edit Mailing List',
 
     'LBL_EMAIL_EMPTYING_TRASH' => 'Emptying Trash',
     'LBL_EMAIL_DELETING_OUTBOUND' => 'Deleteting outbound server',
```

Comment:
see PR https://github.com/salesagility/SuiteCRM/pull/4837 for english typos in 
LBL_EMAIL_DELETING_OUTBOUND
LBL_EMAIL_CLEARING_CACHE_FILES